src/screens/Testing/TestProcess/index.tsx

```diff
@@ -1,44 +1,90 @@
-import React, {useCallback} from 'react';
-import {Timer} from '@src/components/Timer/index';
+import React, {useCallback, useState} from 'react';
+import {Timer} from '@src/components/Timer';
 import {
   ViewContainer,
   ViewFlexRight,
-} from '@src/components/ui/ReadyStyles/Containers/index';
+} from '@src/components/ui/ReadyStyles/Containers';
 import {TimerBox, TextBox, ButtonsBox} from './styles';
-import {MainTestingContainer} from '../../../components/ui/ReadyStyles/Containers';
-import {AnswersOptions} from '@src/components/AnswersOptions/index';
-import {TestOptionsMoc} from '@src/Mocs/Testing';
-import {AppButton} from '@src/components/ui/AppButton/index';
+import {MainTestingContainer} from '@src/components/ui/ReadyStyles/Containers';
+import {AnswersOptions} from '@src/components/AnswersOptions';
+import {AppButton} from '@src/components/ui/AppButton';
 import {ViewFlexCenter, CountQuestionBox, ViewBlock} from './styles';
-import {ProgressBar} from '@src/components/ProgressBar/index';
+import {ProgressBar} from '@src/components/ProgressBar';
+import {ProgressType} from '@src/components/ProgressBar/ProgressView';
+import {useAppSelector} from '@hooks/hooks';
 
 export const TestProcess = () => {
+  const quizIdMocState = useAppSelector(state => state.processReducer);
+  const [numAnswer, setNumAnswer] = useState<number>(1);
+  const currentTest = quizIdMocState.questions.filter(e => e.id === numAnswer);
   const onPressRadioHandler = useCallback((value: number) => {
     console.log(value); // It's temporary
   }, []);
+  const onPressNextAnswer = () => {
+    if (numAnswer < quizIdMocState.questions.length) {
+      setNumAnswer(numAnswer + 1);
+    }
+  };
+  const onPressSkipAnswer = () => {
+    if (numAnswer < quizIdMocState.questions.length) {
+      setNumAnswer(numAnswer + 1);
+    }
+  };
```

Comment:
What is the difference between onPressNextAnswer and onPressSkipAnswer?

Reply:
the logic will be added to onPressSkipAnswer. Haven't decided yet.